vignettes/programming_strategy.Rmd

```diff
@@ -683,50 +695,61 @@ parameters which are not passed as `exprs()` argument (e.g. `new_var = VAR1` or
 
 ## Unit Testing
 
-Unit tests for deprecated functions and parameters must be added to
-`tests/testthat/test-deprecation.R` to ensure that a warning or error is issued. 
+Unit tests for deprecated functions and arguments must be added to the test
+file of the function to ensure that a warning or error is issued.
 
-When writing the unit test, check that the error or warning has the right class, i.e. "lifecycle_error_deprecated" or "lifecycle_warning_deprecated", respectively. The unit-test should follow the corresponding format, per the [unit test guidance](unit_test_guidance.html#writing-unit-tests-in-admiral-):
+When writing the unit test, check that the error or warning has the right class,
+i.e. `"lifecycle_error_deprecated"` or `"lifecycle_warning_deprecated"`,
+respectively. The unit-test should follow the corresponding format, per the
+[unit test guidance](unit_test_guidance.html#writing-unit-tests-in-admiral-).
```

Comment:
Is the hyphen at the end okay?  - guess it clear the link checker

Reply:
Good catch! The hyphen breaks the link partially. I have fixed it.